misc/python/materialize/output_consistency/execution/query_execution_manager.py

```diff
@@ -0,0 +1,251 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+from materialize.output_consistency.common.configuration import (
+    ConsistencyTestConfiguration,
+)
+from materialize.output_consistency.common.format_constants import (
+    COMMENT_PREFIX,
+    CONTENT_SEPARATOR_1,
+)
+from materialize.output_consistency.data_type.data_type_with_values import (
+    DataTypeWithValues,
+)
+from materialize.output_consistency.execution.evaluation_strategy import (
+    EvaluationStrategy,
+)
+from materialize.output_consistency.execution.sql_executor import (
+    SqlExecutionError,
+    SqlExecutor,
+)
+from materialize.output_consistency.execution.test_summary import ConsistencyTestSummary
+from materialize.output_consistency.query.query_format import QueryOutputFormat
+from materialize.output_consistency.query.query_result import (
+    QueryExecution,
+    QueryFailure,
+    QueryResult,
+)
+from materialize.output_consistency.query.query_template import QueryTemplate
+from materialize.output_consistency.validation.result_comparator import ResultComparator
+from materialize.output_consistency.validation.validation_outcome import (
+    ValidationOutcome,
+)
+
+
+class QueryExecutionManager:
+    """Requests the execution of queries and handles transactions"""
+
+    def __init__(
+        self,
+        evaluation_strategies: list[EvaluationStrategy],
```

Comment:
We usually use `List` via `from typing import List` for type hints. I think it mostly matters for old Python version compatibility.

Reply:
✅ Replaced. Done as well for `set` and `dict`.